Demo/Demo/Models/EmptyBodyParams.swift

```diff
@@ -0,0 +1,3 @@
+import Foundation
+
+struct EmptyBodyParams: Codable {}
```

Comment:
We needed to create this to make an empty `POST` request for Order capture and authorize calls.